pandas/core/generic.py

```diff
@@ -9953,11 +9953,24 @@ def _check_percentile(self, q):
         """
 
     @Appender(_shared_docs['pct_change'] % _shared_doc_kwargs)
```

Comment:
Will certainly want to update the docstring to reflect changes being made here. Probably worth outlining logic in a dedicated Notes section